api/notd/api/models_v1.py

```diff
@@ -288,3 +288,7 @@ class ApiSuperCollectionOverlap(BaseModel):
     otherRegistryAddress: str
     otherRegistryTokenCount: int
     registryTokenCountMap: Dict[str, int]
+
+class ApiMintedTokenCount(BaseModel):
+    date: Union[datetime.datetime, datetime.date]
```

Comment:
this should only ever be a datetime.datetime

Reply:
(should be obiovous cos we NEVER have Union[date, datetime] anywhere in a model

Reply:
I did it like this cause depending on the duration, we can either have days ie "2022-12-31" or datetime "2021-02-11T13:00:00" in the response. Or for days should it always be "2022-12-31T00:00:00"

Reply:
yes we store dates as datetimes everywhere so should just return this outwards. optional types are super confusing for whoever is consuming the api